ui/DesignSystem/Markdown.svelte

```diff
@@ -9,6 +9,26 @@
   import marked from "marked";
 
   export let content: string;
+
+  const renderer = {
+    link(href: string, _title: string, text: string) {
```

Comment:
If I’m not mistaken, this does not handle inline html links properly. E.g. if your markdown looks like this

```markdown
[md link](foo)
<a href="bar">inline html</a>
```
I would prefer to style this with CSS. We can even use CSS attribute selector to disable user interaction.

I’d also like to keep the global link styles and disable them for `markdown` with the `:not` selector

Reply:
Inline html works just fine:
```md
[inline link](foo)

[inline link 2](https://twitter.com/)
```

Translates to:

<img width="783" alt="Screenshot 2021-09-20 at 15 45 58" src="https://user-images.githubusercontent.com/158411/134013031-a15e2955-2289-40da-bc00-225d3a6e7fd3.png">

I'm not sure we'll be able to easily achieve the external link styling without additional markup in the specific way that @brandonhaslegs wanted.

How would you distinguish between internal and external links with CSS?

Reply:
> Inline html works just fine:

Your example doesn’t look like it’s using line HTML. I’ve tried the following

```markdown
<a href="https://x">HTML HTTPS</a>

<a href="http://x">HTML HTTP</a>

<a href="./">HTML internal</a>

[Markdown HTTPS](https://x)

[Markdown HTTP](http://x)

[Markdown internal](./foo)
```

and it gives this result

![Screenshot from 2021-09-20 16-45-30](https://user-images.githubusercontent.com/3919579/134022480-42fc3622-d63c-491e-93ec-63619e1f5aa0.png)


> How would you distinguish between internal and external links with CSS?

For example with
```css
  .markdown :global(a[href^="http://"]:after),
  .markdown :global(a[href^="https://"]:after)
  {
    content: "↗";
    margin-left: 0.1rem;
    text-decoration: none;
    /* disables text decoration from containing a element */
    display: inline-block;
  }
```

Not underlining `img` tags might be tricky though

Reply:
Is this what you had in mind https://github.com/radicle-dev/radicle-upstream/commit/258ce958bddeeb1c5eefad0ea737f05ca3d87d70?

```
<a href="https://x">HTML HTTPS</a>

<a href="http://x">HTML HTTP</a>

<a href="./">HTML internal</a>

[Markdown HTTPS](https://x)

[Markdown HTTP](http://x)

[Markdown internal](./foo)
```

Translates to:

<img width="610" alt="Screenshot 2021-09-21 at 11 03 31" src="https://user-images.githubusercontent.com/158411/134143283-6d105f24-368b-4fe4-b025-b952fc333df6.png">

I couldn't figure out how to make the `<img>` tags to not show the `↗` though.

<img width="654" alt="Screenshot 2021-09-21 at 11 04 01" src="https://user-images.githubusercontent.com/158411/134143341-9b397975-ef0a-47f5-bf0f-ec82d9a47ec9.png">

Anchors seem to work, so I left them enabled. Only disabled internal relative links:

<img width="285" alt="Screenshot 2021-09-21 at 11 05 14" src="https://user-images.githubusercontent.com/158411/134143610-bd6c4101-6741-4eb7-95b8-3a4e03a9b64c.png">